### PR TITLE
Update inf-terraform-aws Quickstarter with latest Updates

### PR DESCRIPTION
**Pull request by nichtraunzer**
Updates contains
- #687
- #705 
- #731 
- #734

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,12 @@
 - Jupyter Lab: reduction to a minimal initial env ([#710](https://github.com/opendevstack/ods-quickstarters/issues/710))
 - terraform agent sops/age added ([#730](https://github.com/opendevstack/ods-quickstarters/issues/730))
 - Upgrade python flask quickstarter to Flask 2 version and general dependencies upgrades ([#746](https://github.com/opendevstack/ods-quickstarters/issues/746))
+- Add Node.js 16 builder agent ([#763](https://github.com/opendevstack/ods-quickstarters/issues/763)
+- Update fe-angular to Angular 13.3.0 ([#765](https://github.com/opendevstack/ods-quickstarters/issues/765))
```
Comment (tbugfinder) [inline CHANGELOG.md:15]: above two lines are not part of this PR.

Comment (nichtraunzer) [reply CHANGELOG.md]: good catch - resolved

**Review by tbugfinder — APPROVED** [review]
LGTM
